cogs/birthday/birthday.py

```diff
@@ -145,35 +145,29 @@ async def addMember(self, ctx, member: discord.Member):
             await member.add_roles(role)
         except discord.Forbidden:
             self.logger.error(
-                "Could not add %s#%s (%s) to birthday role, does the bot "
+                "Could not assign %s#%s (%s) to birthday role, does the bot "
                 "have enough permissions?",
                 member.name,
                 member.discriminator,
                 member.id,
                 exc_info=True,
             )
             await ctx.send(
-                ":negative_squared_cross_mark: **Birthday - Add**: Could "
-                "not add **{}** to the list, the bot does not have enough "
+                ":negative_squared_cross_mark: **Birthday - Assign**: Could "
+                "not assign **{}** to the role, the bot does not have enough "
                 "permissions to do so! Please make sure that the bot is "
                 "above the birthday role, and that it has the Manage Roles"
                 "permission!".format(member.name)
             )
             return
 
-        # Save settings
-        async with self.config.member(member).all() as userConfig:
-            userConfig[KEY_IS_ASSIGNED] = True
-            userConfig[KEY_DATE_SET_MONTH] = int(time.strftime("%m"))
-            userConfig[KEY_DATE_SET_DAY] = int(time.strftime("%d"))
-
```

Comment:
I think we need to keep `KEY_IS_ASSIGNED` or else how will the bot remove the role the next day if we don't keep some sort of state?
```suggestion
        await self.config.member(member).userConfig.set(True)
```

Reply:
The `assign` command was only meant to give a user the birthday role. Since I don't see a point of giving someone the role and nor adding them into the config. If you wanted to add them to the config you can just use the `add` command. Was there any main reason for using the `assign` command before?

Reply:
Per our offline discussion before, this was back when we didn't have auto add during the day. If you're refactoring this, then it's probably best to get rid of it, but you should also assign the role if we don't pass in a date in your other command, see other comment.

Reply:
In that case, if you are okay with it. Could I just remove the command as it adds no significant value?

Reply:
Yeah just nuke the command, no need for it anymore.